dask/array/creation.py

```diff
@@ -314,10 +314,12 @@ def meshgrid(*xi, **kwargs):
 
     xi = [e.flatten() for e in xi]
     dimensions = [xi[i].size for i in range(len(xi))]
+    chunks = [xi[i].chunks[0] for i in range(len(xi))]
```

Comment:
What's the reason for taking only the first chunk?

Reply:
NVM forgot that we have to strip the `tuple` enclosing the chunks for the 1-D arrays. This is fine.